pkgs/applications/science/chemistry/gwyddion/default.nix

```diff
@@ -1,36 +1,22 @@
 { lib, stdenv, fetchurl, gtk2, pkg-config, fftw, file,
-  pythonSupport ? false, pythonPackages ? null,
```

Comment:
That is exactly what I am always talking about. Things are just forgotten and left behind in a broken state and no one notices.

Reply:
yeah